pkg/kubelet/cpumanager/cpu_manager.go

```diff
@@ -88,6 +94,20 @@ func NewManager(policyName PolicyName, cr internalapi.RuntimeService, kletGetter
 	}, nil
 }
 
+func getReserverdCores(nodeStatus v1.NodeStatus) int64 {
+	// Kube/System Reserved = Capacity - Allocatable
+	capacity := nodeStatus.Capacity[v1.ResourceCPU]
+	reserved := capacity.Copy()
+	reserved.Sub(nodeStatus.Allocatable[v1.ResourceCPU])
+	glog.Infof("[cpumanager] %v cores reserved for kube/system and shared pool", reserved.Value())
+	if reserved.IsZero() {
+		glog.Infof("[cpumanager] defaulting to 1 core")
```

Comment:
I'm wondering whether we'd be better off dropping this conditional for now... if the operator really wants zero CPUs to be reserved, then there's nothing really "wrong" with it IMO. I think reserving CPU 0 is more of an artifact of the stopgap we had in place before this PR.